ee/consoleuser/consoleuser_windows.go

```diff
@@ -88,16 +90,21 @@ func explorerProcesses(ctx context.Context) ([]*process.Process, error) {
 	return explorerProcs, nil
 }
 
-func processOwnerUid(ctx context.Context, proc *process.Process) (string, error) {
+func processOwnerUid(ctx context.Context, logger log.Logger, proc *process.Process) (string, error) {
 	username, err := proc.UsernameWithContext(ctx)
 	if err != nil {
 		return "", fmt.Errorf("getting process username (for pid %d): %w", proc.Pid, err)
 	}
 
+	// Attempt to lookup the user by userid. This can fail on Windows, but it's not strictly
+	// necessary - we just need something to uniquely identify the user. Fallback to the username
+	// when this happens.
 	user, err := user.Lookup(username)
 	if err != nil {
-		return "", fmt.Errorf("looking up username %s: %w", username, err)
+		level.Debug(logger).Log("msg", "could not lookup windows username", "err", err)
+		return username, nil
```

Comment:
So this changes the semantics here. We were returning a `uid` and now we're returning the `username`. Is it going to be a problem at the point we subsequently exec as the user? Basically, everything over in https://github.com/kolide/launcher/blob/main/ee/desktop/runner/runner.go#L484

Reply:
Especially https://github.com/kolide/launcher/blob/main/ee/desktop/runner/runner_windows.go#L15

Reply:
I think the only time it would be an issue is if we fell back to username ... and then did a check again and got a uid instead. Might be best to just switch to using username across the board on windows.

Reply:
Removing the user lookup entirely will make this change simpler!